roles/ipaclient/vars/Amazon-2.yml

```diff
@@ -0,0 +1,7 @@
+# defaults file for ipaclient
+# vars/RedHat-2
+# Provides support for RedHat-7 based Amazon Linux 2
+
+ipaclient_packages: [ "ipa-client", "libselinux-python" ]
+# The 'ipapython' module is not available in python3.
```

Comment:
Maybe this could be changed to something like in your last comment to point out why this is not the same as in RHEL-7.

Reply:
Updated